crates/bevy_ecs/src/system/system_piping.rs

```diff
@@ -412,10 +423,16 @@ pub mod adapter {
             unimplemented!()
         }
 
+        fn not(In(val): In<bool>) -> bool {
+            !val
+        }
```

Comment:
Maybe move it into `conditions` module? I think it's a useful condition. Unless we going to have an extension trait that `.not()` method.